commit 2aa036d94bffc205c4bc5c20abdb6ad1e87b932c
Author: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
Date:   Wed Oct 17 21:01:56 2018 +0200

    staging: bcm2835-audio: update TODO
    
    commit 01ec7398c56e8f1b903ecb3c5c75400e263eef43 upstream.
    
    The following tasks were completed or not the right solution:
    
    1/2- Not the proper solution, we should register a platform device in
    vchiq the same way it's done with bcm2835-camera as commented here:
    https://lkml.org/lkml/2018/10/16/1131
    
    2/3- Fixed by Takashi Iwai here: https://lkml.org/lkml/2018/9/4/587
    
    Also, adds a new task as per mailing list conversation.
    
    Signed-off-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
    Acked-by: Stefan Wahren <stefan.wahren@i2se.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/TODO b/drivers/staging/vc04_services/bcm2835-audio/TODO
index 73d41fa631ac..cb8ead3e9108 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/TODO
+++ b/drivers/staging/vc04_services/bcm2835-audio/TODO
@@ -4,26 +4,7 @@
 *                                                                           *
 *****************************************************************************
 
+1) Revisit multi-cards options and PCM route mixer control (as per comment
+https://lkml.org/lkml/2018/9/8/200)
 
-1) Document the device tree node
-
-The downstream tree(the tree that the driver was imported from) at
-http://www.github.com/raspberrypi/linux uses this node:
-
-audio: audio {
-	compatible = "brcm,bcm2835-audio";
-	brcm,pwm-channels = <8>;
-};
-
-Since the driver requires the use of VCHIQ, it may be useful to have a link
-in the device tree to the VCHIQ driver.
-
-2) Gracefully handle the case where VCHIQ is missing from the device tree or
-it has not been initialized yet.
-
-3) Review error handling and remove duplicate code.
-
-4) Cleanup the logging mechanism.  The driver should probably be using the
-standard kernel logging mechanisms such as dev_info, dev_dbg, and friends.
-
-5) Fix the remaining checkpatch.pl errors and warnings.
+2) Fix the remaining checkpatch.pl errors and warnings.
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1476_2aa036d94bffc205c4bc5c20abdb6ad1e87b932c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1476_2aa036d94bffc205c4bc5c20abdb6ad1e87b932c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
